### DISCUSSION TITLE
Showcase

### DISCUSSION THREAD

**Discussion by arsham** [rebelot/heirline.nvim#4] [Show and tell]
Thank you for the amazing plugin.

I have [replicated](https://github.com/arsham/arshamiser.nvim) my status line which is using feline, to use this plugin. This is what I found:

- Conversion was straightforward.
- The inheritance model helps reducing duplication.
- Although I still need to refactor the code and reuse more sections, the code almost halved already.
- It has been a couple of hours since you're shared this plugin, but I already can see the potentials.

Thank you for the amazing work!

**Comment by arsham** [comment]
Screenshots:

![1](https://user-images.githubusercontent.com/428611/149636499-da65e533-2378-4538-a731-7f5d940776e3.png)
![2](https://user-images.githubusercontent.com/428611/149636500-3255aed2-026d-4abf-9d99-df05b878541e.png)

**Reply by olimorris** [reply]
This looks awesome. Do you have the config to share?

EDIT: Just seen your [link](https://github.com/arsham/arshamiser.nvim/blob/master/lua/arshamiser/heirliniser/init.lua)

**Reply by Xaxeric** [reply]
cool

**Comment by arsham** [comment]
The experimental branch has improved the elasticity. Here is a sneak pick:

![1](https://user-images.githubusercontent.com/428611/151219806-98df6bde-d52e-4331-af17-557905894774.gif)

**Reply by Zeioth** [reply]
@arsham it looks awesome. Consider PR it to [heirline-components.nvim](https://github.com/Zeioth/heirline-components.nvim) when you have time.
